Resources/Prototypes/SS220/CryopodSSD/CryopodSSD.yml

```
@@ -0,0 +1,50 @@
+﻿- type: entity
+  parent: [ BaseMachinePowered, ConstructibleMachine ]
+  id: SSDStorageConsole
+  name: Консоль крио хранилища
+  description:
+  components:
+    - type: AccessReader
+      access: [ [ "Command" ] ]
+    - type: Appearance
+    - type: Sprite
+      sprite: SS220/Structures/Machines/ssdStorageConsole.rsi
+      layers:
+        - state: ssdStorageConsole
+    - type: Icon
+      sprite: SS220/Structures/Machines/ssdStorageConsole.rsi
+      state: icon
+    - type: Transform
+      anchored: true
+    - type: Storage
+      capacity: 2000000000
+      clickInsert: false
+    - type: ActivatableUI
+      key: enum.SSDStorageConsoleKey.Key
+    - type: UserInterface
+      interfaces:
+        - key: enum.SSDStorageConsoleKey.Key
+          type: CryopodSSDBoundUserInterface
+    - type: SSDStorageConsole
+      whitelist:
+        tags:
+          - HighRiskItem
+
```

```suggestion
    - type: ContainerContainer
      containers:
        machine_board: !type:Container
          showEnts: False
          occludes: True
          ents: []
        machine_parts: !type:Container
          showEnts: False
          occludes: True
          ents: []
        storagebase: !type:Container
          showEnts: False
          occludes: True
          ents: []

```
Нужно добавить ContainerContainer, иначе во время маппинга он будет создан при добавлении комнонента в энтити и затем сохранён в файле карты, а это лишний мусор

---

коммент зафакапился конечно, но суть понятна машин борд и прочее тоже не помешало бы вписать

---

Я решил убрать наследование от машины потому что считаю что крио не должно питаться от энергии или иметь возможность быть разобраным, все таки это выход из игры и он должен быть доступен всегда.
А ContainerContainer, как ты успел заметить я уже успел добавить, так что тыкаю резолв